ui.apps/src/main/content/jcr_root/apps/core/fd/components/form/datepicker/v1/datepicker/clientlibs/site/js/datepickerview.js

```diff
@@ -50,11 +50,19 @@
             return this.element.querySelector(DatePicker.selectors.errorDiv);
         }
 
+        setFocus() {
```

Comment:
`setFocus` should be in BaseFormField. if someone wants, she can override it in the core components.